cypress-tests/cypress/support/commands.js

```diff
@@ -295,12 +354,18 @@ Cypress.Commands.add("captureCallTest", (requestBody, amount_to_capture, payment
     expect(response.headers["content-type"]).to.include("application/json");
     expect(response.body.payment_id).to.equal(payment_id);
     console.log(response.body);
-    if (amount_to_capture == amount) {
+    if (amount_to_capture == amount && response.body.status == "succeeded") {
```

Comment:
```suggestion
    if (amount_to_capture == amount && response.body.status === "succeeded") {
```

Reply:
@pixincreate This is working as expected so why i have to change ?

Reply:
As said before, it is better to have stricter checks and using `===`, `!==` should be preferred over `==`, `!=`.